docs/standard/security/ensuring-data-integrity-with-hash-codes.md

```diff
@@ -1,39 +1,41 @@
-ms.date: "03/30/2017"
+ms.date: 07/14/2020
 ms.technology: dotnet-standard
 dev_langs: 
   - "csharp"
   - "vb"
 helpviewer_keywords: 
   - "generating hash"
   - "verifying hash codes"
-  - "cryptography [.NET Framework], hash"
+  - "cryptography [.NET], hash"
   - "data integrity"
   - "checking hash codes"
-  - "encryption [.NET Framework], hash"
+  - "encryption [.NET], hash"
   - "hash"
 ms.assetid: 33660f33-b70f-4dca-8c87-ab35cfc2961a
 ---
 # Ensuring Data Integrity with Hash Codes
 A hash value is a numeric value of a fixed length that uniquely identifies data. Hash values represent large amounts of data as much smaller numeric values, so they are used with digital signatures. You can sign a hash value more efficiently than signing the larger value. Hash values are also useful for verifying the integrity of data sent through insecure channels. The hash value of received data can be compared to the hash value of data as it was sent to determine whether the data was altered.  
   
- This topic describes how to generate and verify hash codes by using the classes in the <xref:System.Security.Cryptography?displayProperty=nameWithType> namespace.  
+This topic describes how to generate and verify hash codes by using the classes in the <xref:System.Security.Cryptography> namespace.  
   
-## Generating a Hash  
- The managed hash classes can hash either an array of bytes or a managed stream object. The following example uses the SHA1 hash algorithm to create a hash value for a string. The example uses the <xref:System.Text.UnicodeEncoding> class to convert the string into an array of bytes that are hashed by using the <xref:System.Security.Cryptography.SHA1Managed> class. The hash value is then displayed to the console.  
+## Generating a Hash
 
- Due to collision problems with SHA1, Microsoft recommends SHA256 or better.
+ The managed hash classes can hash either an array of bytes or a managed stream object. The following example uses the SHA1 hash algorithm to create a hash value for a string. The example uses the <xref:System.Text.UnicodeEncoding> class to convert the string into an array of bytes that are hashed by using the <xref:System.Security.Cryptography.SHA1> class. The hash value is then displayed to the console.  
+
+ Due to collision problems with SHA1, we recommend SHA256 or better.
   
  [!code-csharp[GeneratingAHash#1](../../../samples/snippets/csharp/VS_Snippets_CLR/generatingahash/cs/program.cs#1)]
  [!code-vb[GeneratingAHash#1](../../../samples/snippets/visualbasic/VS_Snippets_CLR/generatingahash/vb/program.vb#1)]  
   
  This code will display the following string to the console:  
   
- `59 4 248 102 77 97 142 201 210 12 224 93 25 41 100 197 213 134 130 135`  
+ `185 203 236 22 3 228 27 130 87 23 244 15 87 88 14 43 37 61 106 224 81 172 224 211 104 85 194 197 194 25 120 217`  
```

Comment:
If it's printing 32 bytes then presumably it's using SHA256.  Which means the intro paragraph needs to be updated.